static/style.css

```diff
@@ -1,124 +1,122 @@
-html, body, #wrapper{
+html, body, {
     height: 100%;
     width: 100%;
     margin: 0;
 }
 
-#cityMap{
+#mapCanvas{
     height: 100%;
     width: 100%;
     margin: 0;
+    text-align: center;
 }
 
+#navbar{
+  text-align: center;
+  padding: 10px;
+}
 
-ul#logo{
-  position: absolute;
-  top:0px;
-  display: flex;
+a#icon{
+  text-align: center;
+  opacity: 0.7;
 }
 
-ul#logo li{
-  text-align: left;
-  display: block;
+
+a#icon:hover{
   opacity: 1;
 }
 
-ul#navbar{
-  position: absolute;
-  top:0px;
-  margin-left: 350px;
-  display: flex;
+#sidePanel{
+  text-align: left;
+  padding-left: 20px;
+  width: 100%;
 }
 
-
-ul#navbar li{
-  text-align: centre;
-  padding: 0 25px;
-  display: block;
+#about{
+  margin:10px;
+  background-color: #555a66;
   opacity: 0.7;
+  height: 200px;
```

Comment:
Adding height to this div restricts the text in the about description to a very specific length so that it can fit perfectly. It we make it slightly longer, it will get cut off, and if it's too short, it will have a huge leftover gray space below. Instead, I would just keep the padding in #displayInfo / make it a bit larger for top and bottom paddings and let the div adjust to whatever height the text adds up to. We can control how long this description is to make sure it doesn't push down the Stats fun facts too much.

<img width="1392" alt="Screen Shot 2019-05-01 at 12 11 13 AM" src="https://user-images.githubusercontent.com/929246/56999308-89abed80-6ba6-11e9-8c55-56365c038dc2.png">
<img width="1392" alt="Screen Shot 2019-05-01 at 12 13 10 AM" src="https://user-images.githubusercontent.com/929246/56999311-8d3f7480-6ba6-11e9-9a4a-dab14888000e.png">